src/main/java/uk/gov/ons/census/fwmt/outcomeservice/converter/impl/NewUnitAddressLinkedProcessor.java

```diff
@@ -65,6 +65,11 @@ public UUID process(OutcomeSuperSetDto outcome, UUID caseIdHolder, String type)
     root.put("eventDate", eventDateTime);
     root.put("addressType", type);
     root.put("addressLevel", "U");
+    if (type.equals("CE") && outcome.getCeDetails() != null && outcome.getCeDetails().getUsualResidents() != null) {
```

Comment:
can type be null

Reply:
type is passed from the controller survey type, can never be null